inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_nms_5_to_legacy.cpp

```diff
@@ -70,44 +75,49 @@ ngraph::pass::ConvertNMS4ToLegacyMatcher::ConvertNMS4ToLegacyMatcher() {
         }
         auto new_iou_threshold = arg3;
         if (iou_threshold_rank.get_length() == 0) {
-            new_iou_threshold = std::make_shared<ngraph::op::Unsqueeze>(arg3, opset1::Constant::create(element::i64, Shape{1}, {0}));
+            new_iou_threshold = std::make_shared<ngraph::op::Unsqueeze>(arg3, opset1::Constant::create(element::f32, Shape{1}, {0.0f}));
```

Comment:
Why did you change the element type of constant op?

Reply:
You must keep i64 input. This Constant defines the type of the axis to Unsqueeze. It cannot be of float type. Please, revert these changes!

Reply:
Fixed.